plugins/foreman_openscap/1.0/index.md

```diff
@@ -0,0 +1,630 @@
+
+__SCAP Content__ represents SCAP DataStream XML file as defined by SCAP 1.2 standard. DataStream file contains implementation of compliance,
+configuration or security baselines. Users are advised to acquire examplary baseline by installing scap-security-guide package.
+DataStream file usualy contains multiple XCCDF Profiles. Each for different security target. The content of DataStream file can be inspected by oscap tool from openscap-scanner package.
+(XCCDF = Extensible Configuration Checklist Description Format, XCCDF profile = A checklist which audit specific security target)
+
+__Tailoring File__ is a XML file very much like SCAP Content. It represents a customization of compliance defined in a SCAP Content XML file. Additional details and instructions how to create such a file are available in the official openscap [documentation](https://www.open-scap.org/resources/documentation/customizing-scap-security-guide-for-your-use-case/).
+
+__Compliance Policy__ is high level concept of a baseline applied to the infrastructure. Compliance policy is defined by user on web interface. Users may assign following information to the policy:
+
+* Deployment Options - how the client will be deployed. The options are Ansible (provided you have foreman_ansible plugin), Puppet or manual. It is recommended to use Puppet or Ansible for client deployment as they provide a convenient way of applying changes. See section [2.3](plugins/foreman_openscap/0.9/index.html#2.3Policydeploymentoptions) for details on policy deployment options.
+* SCAP Content
+* XCCDF Profile from particular SCAP Content
+* Tailoring File
+* XCCDF Profile from a Tailoring File
+* Hostgroups that should comply with the policy
+* Schedule - the period in which the audit shall occur
+
+__ARF Report__ is XML output of single scan occurrence per single host. Asset Reporting File format is defined by SCAP 1.2 standard. Foreman plug-in stores the ARF Reports in database for later inspections.
+
+
+## 1.2 Release notes
+
+Compatibility matrix
+
+<table class="table table-bordered table-condensed">
+  <tr>
+    <th>Foreman version</th>
+    <th>Plugin version</th>
+    <th>Proxy version</th>
+    <th>Client version</th>
+  </tr>
+  <tr>
+    <td>= 1.7</td>
+    <td>0.3.4</td>
+    <td>0.3.1</td>
+    <td>0.1.1</td>
+  </tr>
+  <tr>
+    <td>>= 1.8</td>
+    <td>0.4.1</td>
+    <td>0.4.0</td>
+    <td>0.1.1</td>
+  </tr>
+  <tr>
+    <td>>= 1.11</td>
+    <td>0.5.0</td>
+    <td>0.5.0</td>
+    <td>0.1.2</td>
+  </tr>
+  <tr>
+    <td>>= 1.13</td>
+    <td>0.6.0</td>
+    <td>0.6.0</td>
+    <td>0.2.0</td>
+  </tr>
+  <tr>
+    <td>>= 1.15</td>
+    <td>0.7.0</td>
+    <td>0.6.0</td>
+    <td>0.3.0</td>
+  </tr>
+  <tr>
+    <td>>= 1.16</td>
+    <td>0.8.0</td>
+    <td>0.6.0</td>
+    <td>0.3.0</td>
+  </tr>
+    <tr>
+    <td>>= 1.17</td>
+    <td>0.9.0</td>
+    <td>>=0.6.0</td>
+    <td>0.3.0</td>
```

Comment:
I guess we could drop old version that are no longer supported, but that is unrelated and can be done separately in future